packages/server/src/Hocuspocus.ts

```diff
@@ -512,7 +511,7 @@ export class Hocuspocus {
       })
     } catch (e) {
       this.closeConnections(documentName)
-      this.documents.delete(documentName)
```

Comment:
I noticed this previously didn't call `document.destroy()` I presume it is safe to do so.